data/scripts/talkactions/commands/force_event.lua

```diff
@@ -21,5 +13,6 @@ function talk.onSay(player, words, param)
 	return returnValue
 end
 
-talk:separator(" ")
-talk:register()
+talkaction:separator(" ")
+talkaction:access(true)
```

Comment:
Missing account type
```suggestion
talkaction:access(true)
talkaction:accountType(ACCOUNT_TYPE_GAMEMASTER)
```

Reply:
this is the same as currently